orttraining/orttraining/python/training/onnxblock/optim/optim.py

```diff
@@ -125,77 +60,207 @@ def build(self, gradients_name: str):
         return cgn_node_output_name
 
 
-class AdamW(onnxblock_module.ForwardBlock):
-    """Builds AdamW optimizer onnxblock for the given training parameters.
+class _OptimizerBase(blocks.Block):
+    def __init__(self):
+        super().__init__()
+
+    def build_optimizer_node(self, input_names, output_name, node_name, node_domain, node_attributes):
```

Comment:
Why not call this function `build` and invoke it by calling super().build(...)?

Reply:
just to make the code more self explanatory, can change it if its crucial.